src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -200,19 +200,17 @@ public override void OnStopActivity(Activity activity, object payload)
 
                 var response = context.Response;
 
-                activity.SetTag(SemanticConventions.AttributeHttpStatusCode, response.StatusCode);
```

Comment:
why is this removed?

Reply:
I revisited the spec and understood that both Span Status and the common attributes (http.status_code) should be set.
I've added the common attribute back. Thanks!